<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDomFrm</class>
 <widget class="QWidget" name="addDomFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加影像</string>
  </property>
  <widget class="QPushButton" name="cancleBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>159</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>16</y>
     <width>651</width>
     <height>131</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>单幅影像</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>5</y>
       <width>621</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>GDAL驱动</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>26</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>影像名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>62</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>选择路径：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="domPathGdalBtn">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>60</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="domNameGdalBtn">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>24</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="selectPathGdalBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择..</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>影像切图</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>5</y>
       <width>621</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>TMS驱动</string>
     </property>
     <widget class="QPushButton" name="selectPathTmsBtn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>60</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择..</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="domNameTmsBtn">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>24</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="domPathTmsBtn">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>60</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>26</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>配置文件：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>62</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>选择路径：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
